### DISCUSSION TITLE
The lotus miner is stuck in the "WaitSeed" state.

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9378] [Storage Provider] [answered]
Lotus miner is frequently stuck in the "WaitSeed" status. It must be restarted to continue working. What is the reason?

5.	2022-09-27 01:41:59 +0800 CST:	[event;sealing.SectorPreCommitBatch]	{"User":{}}
6.	2022-09-27 01:42:01 +0800 CST:	[event;sealing.SectorPreCommitBatchSent]	{"User":{"Message":{"/":"bafy2bzaced5iptz3k43nv7cfdkcxedulre7tgjqt6mkpt5gdpyc4l2ondpqhs"}}}
7.	2022-09-27 01:45:30 +0800 CST:	[event;sealing.SectorPreCommitLanded]	{"User":{"TipSet":[{"/":"bafy2bzacedup3ebxv5zagvwqrz56qweuvxhzmwpftis3grly7lao23klh3g5w"},{"/":"bafy2bzacebmutph254n23nl7gfpbkhhpd36jdcr64nxrckxoqkycds7rmrf6a"},{"/":"bafy2bzacea6ovptcisymydk7okplpzz4l6jv5nvxg2jllhjsmd2exfxmljipu"}]}}

**Comment by Reiers** [comment]
Hi @ChaiWenyu1983 

**WaitSeed** 
The WaitSeed state is a security wait requirement by the network that is initiated between the PreCommit 2 and the start of the Commit 1 phase. It is fixed at 150 epochs, i.e 75 minutes long. At the end of the wait seed state the sector aquires some randomness from the chain."

https://lotus.filecoin.io/storage-providers/get-started/tasks/#waitseed

**Does it take longer that 150 Epochs (75 minutes) ?**

**Comment by ChaiWenyu1983** [comment, marked as answer]
This is the second time that this has happened. Last time, the status of "WaitSeed" lasted a day. I solved this problem by restarting "miner". This time it happened again. The "WaitSeed" status has lasted for ten hours.




柴文煜
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "filecoin-project/lotus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月27日(星期二) 上午7:41
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [filecoin-project/lotus] The lotus miner is stuck in the "WaitSeed" state. (Discussion #9378)





 
Hi @ChaiWenyu1983
 
WaitSeed
 The WaitSeed state is a security wait requirement by the network that is initiated between the PreCommit 2 and the start of the Commit 1 phase. It is fixed at 150 epochs, i.e 75 minutes long. At the end of the wait seed state the sector aquires some randomness from the chain."
 
https://lotus.filecoin.io/storage-providers/get-started/tasks/#waitseed
 
Does it take longer that 150 Epochs (75 minutes) ?

**Reply by Reiers** [reply]
The logs above is not helpful. Can you share a bit more? `lotus-miner sectors status --log 864`
Also, are you in sync? If not, that can cause the delay in WaitSeed. `lotus sync status`?
I can see that you also missed windowPost today, which makes me believe that its either lack of resources or the daemon is struggling to keep sync. Thank you!

**Comment by ChaiWenyu1983** [comment]
As follows:
864
SectorID:	864
Status:		Proving
CIDcommD:	baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq
CIDcommR:	bagboea4b5abcbsso433d4c5tflvo73tgk6xnaffn7zvnov77mc736tyjc6temakz
Ticket:		95b8db9ccaea1c867c9caa250c0609c50914dc23be34a57602eb8abdd119d3cd
TicketH:	2195647
Seed:		a336b1f4d43766c7c113a6a3bed9463216a4e22f70f5fde9129237da93783b79
SeedH:		2197075
Precommit:	bafy2bzaced5iptz3k43nv7cfdkcxedulre7tgjqt6mkpt5gdpyc4l2ondpqhs
Commit:		bafy2bzaceabkgke6z4bf4ily44yjjcfvr6wlmjizdbwlkewq4p6g35mahbvck
Deals:		[0]
Retries:	0
--------
Event Log:
0.	2022-09-26 22:30:00 +0800 CST:	[event;sealing.SectorStartCC]	{"User":{"ID":864,"SectorType":8}}
1.	2022-09-26 22:33:45 +0800 CST:	[event;sealing.SectorPacked]	{"User":{"FillerPieces":[{"Size":34359738368,"PieceCID":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}]}}
2.	2022-09-26 22:33:46 +0800 CST:	[event;sealing.SectorTicket]	{"User":{"TicketValue":"lbjbnMrqHIZ8nKolDAYJxQkU3CO+NKV2AuuKvdEZ080=","TicketEpoch":2195647}}
3.	2022-09-27 01:29:55 +0800 CST:	[event;sealing.SectorPreCommit1]	{"User":{"PreCommit1Out":"eyJfbG90dXNfU2VhbFJhbmRvbW5lc3MiOiJsYmpibk1ycUhJWjhuS29sREFZSnhRa1UzQ08rTktWMkF1dUt2ZEVaMDgwPSIsImNvbW1fZCI6WzcsMTI2LDk1LDIyMiw1MywxOTcsMTAsMTQ3LDMsMTY1LDgwLDksMjI3LDczLDEzOCw3OCwxOTAsMjIzLDI0MywxNTYsNjYsMTgzLDE2LDE4Myw0OCwyMTYsMjM2LDEyMiwxOTksMTc1LDE2Niw2Ml0sImNvbmZpZyI6eyJpZCI6InRyZWUtZCIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC04NjQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjIxNDc0ODM2NDd9LCJsYWJlbHMiOnsiU3RhY2tlZERyZzMyR2lCVjEiOnsiX2giOm51bGwsImxhYmVscyI6W3siaWQiOiJsYXllci0xIiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTg2NCIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTIiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtODY0Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItMyIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC04NjQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci00IiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTg2NCIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTUiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtODY0Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItNiIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC04NjQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci03IiwicGF0aCI6Ii9tbnQvc3NkMy9zZWFsaW5nL2NhY2hlL3MtdDAxODYyNTY0LTg2NCIsInJvd3NfdG9fZGlzY2FyZCI6Nywic2l6ZSI6MTA3Mzc0MTgyNH0seyJpZCI6ImxheWVyLTgiLCJwYXRoIjoiL21udC9zc2QzL3NlYWxpbmcvY2FjaGUvcy10MDE4NjI1NjQtODY0Iiwicm93c190b19kaXNjYXJkIjo3LCJzaXplIjoxMDczNzQxODI0fSx7ImlkIjoibGF5ZXItOSIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC04NjQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci0xMCIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC04NjQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9LHsiaWQiOiJsYXllci0xMSIsInBhdGgiOiIvbW50L3NzZDMvc2VhbGluZy9jYWNoZS9zLXQwMTg2MjU2NC04NjQiLCJyb3dzX3RvX2Rpc2NhcmQiOjcsInNpemUiOjEwNzM3NDE4MjR9XX19LCJyZWdpc3RlcmVkX3Byb29mIjoiU3RhY2tlZERyZzMyR2lCVjFfMSJ9"}}
4.	2022-09-27 01:41:59 +0800 CST:	[event;sealing.SectorPreCommit2]	{"User":{"Sealed":{"/":"bagboea4b5abcbsso433d4c5tflvo73tgk6xnaffn7zvnov77mc736tyjc6temakz"},"Unsealed":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}}
5.	2022-09-27 01:41:59 +0800 CST:	[event;sealing.SectorPreCommitBatch]	{"User":{}}
6.	2022-09-27 01:42:01 +0800 CST:	[event;sealing.SectorPreCommitBatchSent]	{"User":{"Message":{"/":"bafy2bzaced5iptz3k43nv7cfdkcxedulre7tgjqt6mkpt5gdpyc4l2ondpqhs"}}}
7.	2022-09-27 01:45:30 +0800 CST:	[event;sealing.SectorPreCommitLanded]	{"User":{"TipSet":[{"/":"bafy2bzacedup3ebxv5zagvwqrz56qweuvxhzmwpftis3grly7lao23klh3g5w"},{"/":"bafy2bzacebmutph254n23nl7gfpbkhhpd36jdcr64nxrckxoqkycds7rmrf6a"},{"/":"bafy2bzacea6ovptcisymydk7okplpzz4l6jv5nvxg2jllhjsmd2exfxmljipu"}]}}
8.	2022-09-27 14:28:50 +0800 CST:	[event;sealing.SectorSeedReady]	{"User":{"SeedValue":"ozax9NQ3ZsfBE6ajvtlGMhak4i9w9f3pEpI32pN4O3k=","SeedEpoch":2197075}}
9.	2022-09-27 15:58:27 +0800 CST:	[event;sealing.SectorCommitted]	{"User":{"Proof":"hSclbU512T7lE6B8iXO8+TVCV9nbnhhrVgkau+iyjvR1ua6SPpHNG9cE9Z7XwZJKrGSCnGvJfdF1A3kEkMypYr0bB38WS+WclVffGmahcGEYbZQGyqOq+PVFssdcm0VkDOZ/tPF9G36SyGWQShxcOCihc0qhb78Thrkwl5csQrbyMZQyXNkYNFLQ1LZcmJ+YpDQDTyFSKeGkfXNjCbEX9o/Ghi7VIsUdBOcjCHmzIn5L6fO+0+yYFdU4g7OEnJkekbm4jfsIPN/OlkbK60SxJNVcSQu5NCSsYJqx3tI9xlTXsTxxPqLfUy/LrWFtj1ZEhqP/KeuBZUpnl3idXXlpsyXKwPHXNkXLl/itm2GEbpukr8pNWiA+XzdlHrufm82jCbhQznsblZpWg+t14CZl6FDQBz5ysKeumUvcycyXRWeORkzBUyiQfJGRQlKrsng4hxhOluNT/GLNor0+tHJze8W1AOKYhQu29BgEjyAuGqrdO1KnJEzIUL8e3t4mllVeudGSYc/EDZlFMAdWt8eEZU5irEjpZeuys5PhfoI3IA/GC4oqC9gbFDT/0gF4qkzQhfTboFY1JI0HyAmFZjkhjXfK23i6Zk9ttmxCVENgFstL+H0fBt5AQSu5pxFPUVe2FRrCG6ynNO+WnKfP8L9hsVLKLhql20l2u6XpbtY5ImE5zgx9cZLc6isJEdMlMJPCp8B1Qyt8UJvBe/8Wlb8SLnfOgcIQd6vDl7yUs6BlAOeRkgauxLm+/2VWSK8kfhVYh4UU+kScY/m8zs3T41IySqhkY8yxfaktQ+dSL4yWmyrImc/yh5WuVMUQY5LJ/BImud0wOKGy614P46ZIMnwEebK6CFutSEczs3zmPwBXeazuQ+9YIoXPO9gimiKUdVAnEsA7gEk0+2wqWfdx11oS5GP1aeUs8L+Qgj55FRnZ+FolMJPwnihdaijTBDvTGlDOhobyEz1h6xRH+8zeTD5IVFsoqulSy1iNwEHHWjGF4pqfMt1wsb57q7WrsCfRrPQygrAn+NiINZ4bkoxiPei9uWDS61EGHVW93UaCpM3YzdNF31cMQ5RHASw1ZiXSxILnohCKmbQdZgg67VQkzph/ux+yRwfJaLQzhIVPM5n6KAb0Zdn3A3f6Ftk0XhI49pZOCTTP1/a+sWEQ2RFDwE/rnUzLNlFP4wdlCELK7AGgRzAvcPkUmADpgAd9D9VO9XqfiT+vOEbfb5hNwp8KtlWwc9bXkEDQ0mKkuB7Eg8GDTHIJ9LYDolzEsruuEcKZBnvagCuTPTAdqU1QE0TNZVXG0oo1dEI9HFALZ/CA7w/A0HpYHnhkJll32duURAzrX6rfgk/PAgYgxupLpk9kr99icgh3rX975qqIZNVm1R9cdFsFvYPToFKriHvQum9ccPrDBCxw252PsgqOFp324Uja+nJ6CvBcQPvk15zbkUGkTWmD+An2Fvlnmkyt3/WH5DDggtdeoGvBCJTCAb9UrXi6YaiEDyhwXgeRmrbv9cBO7KgiiwZopYoEXFz0/NxRTRC/oRBad0P+sXhop3vA4IUs/d3b9BafncELBLFdkyZvcS6SSKf8aoPWMGbTyxdhDv4qrQ4vABdjZM931eKwJ1ahGFVS/5I/j3jxbGm1KFK7cM6C7ftzuVriyYcU05jvYDNsEzRtsTxN3Ahz8bBEMZZEFj+/mH+pZne9Ctro2b9sGt3W0tDTAfaHbCpZlRrHoMf6r4V00fb91zcHtnZp35pBrRbYi4PJbPGuFEGkk2tDzQPMPRWHj5mrXYAKBLgAfHyJmW5dHb9hlO97vBnyNgPwcAuIQCqeBwLTFE/X+r+QwwDvsNa/ITfVxWsrDM8oX74TiB+SOgYYUeXxP9zmfWug/VOU1VnHsnez5P1euyiQh5xP/Dcnq9VKTzBWXpKqQ6aKDEOtIUs9ErZ8J2qwfxsyMymboTcaWd0PHJvF276n47gKZXjPC/CQa/TmuBVNc/MBsFVUaMjuU0Bk68TDgbp3Xx7kvvmlJFibw9ZBpSIjNcXDezDm1tRKANQuSItCJ7Ewpk6LkowJBtKc6thEI7xW0FIrMMuVSFfh79Q5up1bqCRJtcR9QtK/UYNp2fFi3TT5ufo4ph8Artwlod9M4exZRLvV0x9V9LA4pmfHLVpSpW0V1oth+CXI01Ua8NTUERErFvH1sBUnEvD40c3nGdIoGfURijRqlIAWtgkp+0ua+yBzeI89/hslV4BWioueOWknl9Yj208nnXaSXXpcQgr/mffpjXfD2SHpFncRQgvvoRiJ+MevWpkyOpHvQs+OuYZvl1uCDAtyEcZ+MB4uwo4b3HQBXC87Fd8Dmlfs46viJmxF0g7l/vv+OHAYIwz5NxXQuNjmdAeOWhiTTSiNNQYGIeXljIUVHrbZCo9+IKgzPYVTXV9YGqC8SKQIWL0SGlBFFSCq0g+ucxO8thPYZd3Pq5Gu2r1d84b4zZe9fcynfa0k0jCsxRQKH0fAAYInx4tLlTGaSCe9khgEEXAQG5mpfqkxd3zj064mpZAl0T+erTxZRiggqZPvDfyHRBj/TR8I"}}
10.	2022-09-27 15:58:27 +0800 CST:	[event;sealing.SectorSubmitCommitAggregate]	{"User":{}}
11.	2022-09-27 16:00:02 +0800 CST:	[event;sealing.SectorCommitAggregateSent]	{"User":{"Message":{"/":"bafy2bzaceabkgke6z4bf4ily44yjjcfvr6wlmjizdbwlkewq4p6g35mahbvck"}}}
12.	2022-09-27 16:03:30 +0800 CST:	[event;sealing.SectorProving]	{"User":{}}
13.	2022-09-27 16:05:05 +0800 CST:	[event;sealing.SectorFinalized]	{"User":{}}







柴文煜
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "filecoin-project/lotus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月27日(星期二) 下午5:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [filecoin-project/lotus] The lotus miner is stuck in the "WaitSeed" state. (Discussion #9378)





 
The logs above is not helpful. Can you share a bit more? lotus-miner sectors status --log 864
 Also, are you in sync? If not, that can cause the delay in WaitSeed. lotus sync status?
 I can see that you also missed windowPost today, which makes me believe that its either lack of resources or the daemon is struggling to keep sync. Thank you!

**Reply by Reiers** [reply]
Thanks, I can see the time difference - what `lotus version` are you on? 
Is hard troubleshoot without a full picture, so :

- How much are you sealing?
- Are you in sync? how many peers are you connected to?
`lotus net bandwidth; lotus net peers | wc -l` and `lotus sync status`
- How many sector do you batch together?
- Why did you loose windowPost? Any stuck messages?
`lotus mpool pending --local`

**Comment by ChaiWenyu1983** [comment]
The answer is as follows:
Local: lotus version 1.17.1+mainnet+git.8db6a939c.dirty;

It has been sealed for nearly half a year, and it has been stable. Recently, it began to break down frequently.
At that time, the synchronization check was as good as usual, with more than 170 peers.
It is stable to process three sectors at the same time.






柴文煜
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "filecoin-project/lotus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月27日(星期二) 晚上7:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [filecoin-project/lotus] The lotus miner is stuck in the "WaitSeed" state. (Discussion #9378)





 
Thanks, I can see the time difference - what lotus version are you on?
 Is hard troubleshoot without a full picture, so :
  
How much are you sealing?
 
Are you in sync? how many peers are you connected to?
 lotus net bandwidth; lotus net peers | wc -l and lotus sync status
 
How many sector do you batch together?
 
Why did you loose windowPost? Any stuck messages?
 lotus mpool pending --local

**Comment by ChaiWenyu1983** [comment]
Run in command line mode instead of service mode. It can be closed if it is normal.

**Comment by iseriser** [comment]
Today I was experiencing the same problem:
`	WaitSeed: 44`
`$lotus-miner --version`
`lotus-miner version 1.17.1+mainnet+git.e3894de61`
`$boostd --version`
`boostd version 1.4.0+git.d5d367f`

Miner restart moved all 44 sectors to committing state

**Comment by ChaiWenyu1983** [comment]
At present, it can only be found that the miner solution is restarted after being stuck.在 2022年11月16日，16:58，iseriser ***@***.***> 写道：﻿
Today I was experiencing the same problem:
	WaitSeed: 44
$lotus-miner --version lotus-miner version 1.17.1+mainnet+git.e3894de61
